robot.go

```diff
@@ -10,14 +10,15 @@ import (
 
 // Robot receives messages from an adapter and sends them to listeners
 type Robot struct {
-	Name       string
-	Alias      string
-	Adapter    Adapter
-	Store      Store
-	handlers   []handler
-	Users      *UserMap
-	Auth       *Auth
-	signalChan chan os.Signal
+	Name           string
```

Comment:
Is this alignment imposed by the formatter? It doesn't help when diffing.

Reply:
Yes, `go fmt` for the good and for the bad.

Reply:
There were long discussions regarding this kind of alignment in the Elixir formatter, so I know the story.